src/Umbraco.Web.UI.Client/src/packages/core/server/server-connection.ts

```diff
@@ -86,5 +89,6 @@ export class UmbServerConnection extends UmbControllerBase {
 		this.#versionCheckPeriod.setValue(data?.versionCheckPeriod);
 		this.#allowLocalLogin.setValue(data?.allowLocalLogin ?? false);
 		this.#allowPasswordReset.setValue(data?.allowPasswordReset ?? false);
+		this.#enableMediaRecycleBinProtection.setValue(data?.enableMediaRecycleBinProtection ?? false);
```

Comment:
Could this be moved to some media configuration endpoint instead? It appears that a few things have slipped into this context that probably should have lived in their own silos.